Makefile

```diff
@@ -70,16 +81,29 @@ test:
 	$(call RUN,$(ENV_OPTS) sbt compile test)
 
 dist: COMMIT=$(shell git rev-parse --short=12 --verify HEAD)
-dist: VERSION_FILE=dist/toree-kernel/VERSION
+dist: VERSION_FILE=dist/toree/VERSION
 dist: kernel/target/scala-2.10/$(ASSEMBLY_JAR) ${shell find ./etc/bin/*}
-	@mkdir -p dist/toree-kernel/bin dist/toree-kernel/lib
-	@cp -r etc/bin/* dist/toree-kernel/bin/.
-	@cp kernel/target/scala-2.10/$(ASSEMBLY_JAR) dist/toree-kernel/lib/.
-	@echo "VERSION: $(FULL_VERSION)" > $(VERSION_FILE)
+	@mkdir -p dist/toree/bin dist/toree/lib
+	@cp -r etc/bin/* dist/toree/bin/.
+	@cp kernel/target/scala-2.10/$(ASSEMBLY_JAR) dist/toree/lib/.
+	@echo "VERSION: $(VERSION)" > $(VERSION_FILE)
 	@echo "COMMIT: $(COMMIT)" >> $(VERSION_FILE)
```

Comment:
It would be a good idea to include the commit information into the build info of the kernel: https://github.com/apache/incubator-toree/blob/master/project/Build.scala#L181